lib/libzrepl/data_conn.c

```diff
@@ -1476,6 +1476,7 @@ uzfs_zvol_rebuild_scanner(void *arg)
 				LOG_INFO("Got a snapshot while switching "
 				    "to ALL_SNAP_DONE command for zv:%s",
 				    snap_zv->zv_name);
+				all_snap_done = B_FALSE;
```

Comment:
:+1: 
will it be good to add a VERIFY such that snap_name consists of internal_snapshot string at the `exit`?
and, above TODO message need to be fixed

Reply:
done.